postgres/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.6'
```

Comment:
Can we just name the folder `db` and mount the data in the root... or are you concerned that then this file would then get mounted as well?  If so, what about the folder names `db` and `data` so that the directory structure is `db/data`?

Reply:
Yeah you could, but I like to keep the data separate from source files (i.e. the docker-compose file) for the sake of self-documentation. Also, in the future I would add a README and possibly other assets in the postgres folder, like possibly a seed file. Your call though.